docs/connector-development/connector-builder-ui/authentication.md

```diff
@@ -0,0 +1,150 @@
+# Authentication
+
+Authentication allows the connection to check whether it has sufficient permission to fetch data. The authentication feature provides a secure way to configure authentication using a variety of methods.
+
+The credentials itself (e.g. username and password) are _not_ specified as part of the connector, instead they are part of the configuration that is specified by the end user when setting up a source based on the connector. During development, it's possible to provide testing credentials in the "Testing values" menu, but those are not saved along with the connector. Credentials that are part of the source configuration are stored in a secure way in your Airbyte instance while the connector configuration is saved in the regular database.
+
+In the "Authentication" section on the "Global Configuration" page in the connector builder, the authentication method can be specified. This configuration is shared for all streams - it's not possible to use different authentication methods for different streams in the same connector. In case your API uses multiple or custom authentication methods, you can use the [low-code CDK](/connector-development/config-based/low-code-cdk-overview) or [Python CDK](/connector-development/cdk-python/).
+
+If your API doesn't need authentication, leave it set at "No auth". This means the connector will be able to make requests to the API without providing any credentials which might be the case for some public open APIs or private APIs only available in local networks.
+
+## Authentication methods
+
+Check the documentation of the API you want to integrate for the used authentication method. The following ones are supported in the connector builder:
+* [Basic HTTP](#basic-http)
+* [Bearer Token](#bearer-token)
+* [API Key](#api-key)
+* [OAuth](#oauth)
+
+Select the matching authentication method for your API and check the sections below for more information about individual methods.
+
+### Basic HTTP
+
+If requests are authenticated using the Basic HTTP authentication method, the documentation page will likely contain one of the following keywords:
+- "Basic Auth"
+- "Basic HTTP"
+- "Authorization: Basic"
+- "Base64"
+
+The Basic HTTP authentication method is a standard and doesn't require any further configuration. Username and password are set via "Testing values" in the connector builder and by the end user when configuring this connector as a Source.
+
+#### Example
+
+The [Greenhouse API](https://developers.greenhouse.io/harvest.html#introduction) is an API using basic authentication.
```

Comment:
Nit, we can actually link directly to the `authentication` section in the URL instead of `introduction`:
https://developers.greenhouse.io/harvest.html#authentication